openfl/federated/task/runner_fa.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2020-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+"""
+Base classes for Federated Analytics.
+
+You may copy this file as the starting point of your own keras model.
+"""
+
+from openfl.federated.task.runner import TaskRunner
+
+
+class FederatedAnalyticsTaskRunner(TaskRunner):
```

Comment:
Given the inheritance, I don't think we need to inherit all methods as well, especially when all are passthroughs. It is implied...